src/containers/RepositoriesContainer/RepositoriesDucks.js

```diff
@@ -0,0 +1,37 @@
+// Actions
+const LOAD = 'react-redux-starter-app/repositories/LOAD';
+const LOAD_SUCCESS = 'react-redux-starter-app/repositories/LOAD_SUCCESS';
+const LOAD_FAIL = 'react-redux-starter-app/repositories/LOAD_FAIL';
```

Comment:
They look grand. Your eyes only notice the capitals tbh.

The path here is the Ducks convention, the idea being to namespace your actions. Without our package name, our actions could end up with the same names as those from another package altogether, making the 2 incompatible for users.

They're kind of a pain to set up, but once you do it, you just refer to the constants from then on (And it's nice being able to use LOAD rather than LOAD_REPOSITORIES etc. in the rest of your code too. The extra code is confined to this one section)